templates/components/engines/typescript/agent/chat.ts

```diff
@@ -5,8 +5,39 @@ import path from "node:path";
 import { getDataSource } from "./index";
 import { STORAGE_CACHE_DIR } from "./shared";
 
+enum FunctionToolType {
+  MATH = "math",
+  WEATHER = "weather",
+}
+
+async function importFunctionTools(
```

Comment:
just import everything from the templates/components/engines/typescript/agent/tools folder (which should be automatically copied)